cp/cp_config.c

```diff
@@ -353,8 +353,8 @@ fetch_s1u_sgw_ip(uint32_t dpId)
 		}
 	}
 
-	rte_panic("Could not find s1u ip address for dpid: %u\n", dpId);
-	rte_exit(EXIT_FAILURE, "Could not find s1u ip address for dpid: %u\n", dpId);
+	rte_panic("Could not find s1u ip address for an unregistered DP\n");
+	rte_exit(EXIT_FAILURE, "Could not find s1u ip address for an unregistered DP\n");
```

Comment:
panic would already exit. Also we should not exit. Unless its in bootstrap phase CP should not panic/exit when DPs connect